### PR TITLE
DBZ-3816 Workaround for Kafka Connect bug

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-3816

### CODE REVIEW

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresDefaultValueConverter.java [hunk 1]
```diff
@@ -76,6 +77,11 @@
         try {
             Object rawDefaultValue = mapper.parse(defaultValue);
             Object convertedDefaultValue = convertDefaultValue(rawDefaultValue, column);
+            if (convertedDefaultValue instanceof Struct) {
+                // Workaround for DBZ-3816
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresDefaultValueConverter.java:81]: Did you mean to link to the AK upstream issue here instead?

Comment (jpechane) [reply debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresDefaultValueConverter.java]: @gunnarmorling Updated

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresSchemaIT.java [hunk 2]
```diff
@@ -478,6 +479,7 @@ public void shouldProperlyGetDefaultColumnValues() throws Exception {
             assertColumnDefault("jsonb", "{}", columns);
 
             assertColumnDefault("numeric", new BigDecimal("12345.67891"), columns);
+            assertColumnDefault("numeric_var", null, columns);
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresSchemaIT.java:482]:
```suggestion
            // KAFKA-12694: default value for Struct currently exported as null
            assertColumnDefault("numeric_var", null, columns);
```

File: debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresDefaultValueConverter.java [hunk 3]
```diff
@@ -76,6 +77,11 @@
         try {
             Object rawDefaultValue = mapper.parse(defaultValue);
             Object convertedDefaultValue = convertDefaultValue(rawDefaultValue, column);
+            if (convertedDefaultValue instanceof Struct) {
+                // Workaround for KAFKA-12694
+                LOGGER.warn("Mapper for column '{}' returns Struct which is not supported.", column.name());
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresDefaultValueConverter.java:82]:
```suggestion
                LOGGER.warn("Struct can't be used as default value for column '{}', will use null instead.", column.name());
```

**Review by gunnarmorling — APPROVED** [review]
LGTM, will apply my wording changes and merge. Thx, @jpechane!

**Pull request merged**